cfgov/v1/handlers/blocks/feedback.py

```diff
@@ -49,7 +49,7 @@ def get_response(self, form):
         if form.is_valid():
             feedback = form.save(commit=False)
             is_helpful = self.request.POST.get('is_helpful', None)
-            if is_helpful is not None:
+            if is_helpful is not None and is_helpful != '':
```

Comment:
You can cause a 500 by coming to the page with an invalid query string, e.g. `?is_helpful=text`.

What about using something like this instead:

```py
try:
    feedback.is_helpful = bool(int(is_helpful))
except ValueError:
    pass
```